lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -203,6 +162,103 @@ sub _files_to_categories {
     return $ret;
 }
 
+my $rt_cpan_base = 'https://rt.cpan.org/Ticket/Display.html?id=';
```

Comment:
Actually I misread your comment.  I was considering moving this into ReleaseInfo, but I wasn't certain yet if I wanted to do that.  But it can also be addressed separately.